pkg/cmd/internal/issues/issues.go

```diff
@@ -492,18 +493,19 @@ func HelpCommandAsLink(title, href string) func(r *Renderer) {
 	}
 }
 
-// filterByExactTitleMatch filters the search result passed and
-// removes any issues where the title does not match the expected
-// title exactly. This is done because the GitHub API does not support
-// searching by exact title; as a consequence, without this function,
-// there is a chance we would group together test failures for two
-// similarly named tests. That is confusing and undesirable behavior.
-func filterByExactTitleMatch(
+// filterByPrefixTitleMatch filters the search result passed and removes any
+// issues where the title does not match the expected title, optionally followed
+// by whitespace. This is done because the GitHub API does not support searching
+// by exact title; as a consequence, without this function, there is a chance we
+// would group together test failures for two similarly named tests. That is
+// confusing and undesirable behavior.
+func filterByPrefixTitleMatch(
 	result *github.IssuesSearchResult, expectedTitle string,
 ) []github.Issue {
+	expectedTitleRegex := regexp.MustCompile(`^` + expectedTitle + `(\s+|$)`)
```

Comment:
It just occurred to me: shouldn't we `regexp.QuoteMeta(expectedTitle)`? I don't _think_ this is an immediate problem given our test names and title function, but could become a silent bug if those were to change. What's worse, `MustCompile` would crash the entire roachtest nightly run.

Reply:
yeah that's definitely a good idea